src/inmanta/const.py

```diff
@@ -168,20 +168,46 @@ class LogLevel(Enum):
     TRACE = 3
     NOTSET = 0
 
+    @property
+    def to_str(self) -> "StrLogLevel":
+        """
+        Convert an integer formated log level into a string formated log level
+        """
+        return StrLogLevel[self.name]
+
+
+@stable_api
```

Comment:
If this becomes part of the stable API, could you add it to the docs as well? See [here](https://github.com/inmanta/inmanta-core/blame/master/docs/reference/api.rst#L14-L17) (not sure how to link to a rst source outside of blame view). Depending on whether the methods are considered part of the stable API, you could additionally add them for both classes (add their name after `:members:`.